lib/src/main/java/com/diffplug/spotless/npm/PrettierFormatterStep.java

```diff
@@ -36,11 +37,18 @@
 
 	public static final String NAME = "prettier-format";
 
-	public static FormatterStep create(Provisioner provisioner, File buildDir, @Nullable File npm, PrettierConfig prettierConfig) {
```

Comment:
Since this is `public static`, let's keep it for backward compatibility but add an `@Deprecated` and delegate to the new method.